api/v3/CivisocialUser.php

```diff
@@ -38,7 +38,17 @@ function civicrm_api3_civisocial_user_createContact($params) {
   return CRM_Civisocial_BAO_CivisocialUser::createContact($params);
 }
 
+/**
+ * Fetches Facebook event information
+ *
+ * @param array $params
+ *
+ * @return array
+ *   Array of Facebook event information or error messgaes
+ */
 function civicrm_api3_civisocial_user_getFacebookEventInfo($params) {
+  civicrm_api3_verify_mandatory($params, NULL, array('event_id'));
```

Comment:
I would prefer to use _civicrm_api3_civisocial_user_getFacebookEventInfo_spec(&$params). This is a clean way to document params. And shows much nicer in http://domain.com/civicrm/api

You can define default, required, type etc for each parameter.


You can refer at https://github.com/civicrm/civicrm-core/blob/master/api/v3/Contact.php#L135
Most of api have _spec defined.